src/components/TableBody.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+class TableBody extends React.Component {
+	constructor(props){
+		super(props);
+
+	}
+	render() {
+		const data = this.props.fetched;
+
+		let rowWithData = data.map(function(item) {
+			return <tr>
```

Comment:
Just to be on the safe side, I wrap multi line returns like this in parens `(` and `)`

Reply:
done!